### ISSUE TITLE
Query for running jobs

### ISSUE THREAD

**Issue by Vortec4800** [agenda/agenda#687]
How do you get a list of running jobs? I tried `let runningJobs = await agenda.jobs({ running: 1 });` but that didn't return anything. When the server starts, I want to clear out any jobs that were running previously, in case it didn't shut down cleanly, so they don't clog up the queue.

**Comment by wootwoot1234** [comment]
Did you ever find a solution to this?  I'm having this issue too.

**Comment by Vortec4800** [comment]
I did not, though I found out my specific use-case (removing orphaned jobs) wasn't really necessary, the timeout value takes care of this automatically.

**Comment by wootwoot1234** [comment]
Thanks for writing back.  Do you mean you didn't need to look for orphaned jobs because they were recurring and would eventually be called again?  I just want to make sure I'm understanding how agenda works.  :)

**Comment by Vortec4800** [comment]
Not exactly. When the job run starts, it gets a timeout value set, I seem to recall the default value is 10 minutes. If the server is shut down before the job finishes and then restarted, the job will sit until the timeout expires, then agenda will restart that job and it will run again. This happens regardless of if the job is set to recur or if it's a single-fire job.

I'm leaving this issue open just because while I didn't end up needing this functionality, as far as I can tell there is still no way to get a list of jobs by state, which could be helpful for other purposes.

**Comment by wootwoot1234** [comment]
Agreed, getting the jobs status would be very helpful.
